scripts/ci/get-node-version.mjs

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env node
+import fetch from 'node-fetch'
+import { resolve } from 'path'
+
+import { ROOT, getPackageJSON } from './_common.mjs'
+
+const nodeVersion = await getPackageVersion()
+const version = await getVersion(nodeVersion)
+if (!version) {
+  process.stderr.write(`Failed getting docker image for ${nodeVersion}`)
+  process.exit(1)
+}
+process.stdout.write(version)
+
+async function getVersion(
+  version,
+  withAlpine = true,
+  architecture = 'amd64',
+  url = null,
+) {
+  try {
+    const baseURL =
+      url ??
+      'https://hub.docker.com/v2/repositories/library/node/tags?page_size=100'
```

Comment:
can we move this to a constant at the top of the file?

Reply:
Cool. Will do.